entries/i/ing.de.json

```diff
@@ -1,12 +1,18 @@
 {
   "ING-DiBa Deutschland": {
     "domain": "ing.de",
-    "url": "https://www.ing.de",
     "img": "ing.svg",
     "tfa": [
-      "sms"
+      "custom-software",
+      "custom-hardware"
     ],
-    "documentation": "https://www.ing.de/kundenservice/sicherheit/psd2/",
+    "custom-software": [
+      "ING Banking to go"
+    ],
+    "custom-hardware": [
+      "photoTAN reader"
+    ],
+    "documentation": "https://www.ing.de/hilfe/psd2/",
```

Comment:
Does the documentation give any useful 2FA information in your opinion? I can't find anything useful but I am using Google Translate so I might be wrong.

Reply:
I've changed the link to a more informative one.